CMake/ResolveDependency.cmake

```diff
@@ -111,7 +111,7 @@ function(set_with_default var_name envvar_name default)
 endfunction()
 
 # List subdirectories of ${dir}
-function(list_subdirs var dir)
+function(velox_list_subdirs var dir)
```

Comment:
Seems this function is useless now. Can we remove it? cc @assignUser

Reply:
I don't see any use of it within the `velox` code base, it seems like it would be a good candidate for removal. Happy to take care of it in this PR if you all agree.